universal-application-tool-0.0.1/app/controllers/admin/AdminProgramController.java

```diff
@@ -40,14 +41,17 @@ public AdminProgramController(
     this.formFactory = checkNotNull(formFactory);
   }
 
+  @Secure(authorizers = "uatadmin")
```

Comment:
(blocking): can we turn these into constants from the securitymodule so we don't have strings in the wild?

Reply:
Done.